docs/julia.rst

```diff
@@ -320,168 +320,169 @@ The following functions must be available:
++---------------------------------------------+-----------------------------------------------------------------+
 | create(v:u256, p:u256, s:u256)              | create new contract with code mem[p..(p+s)) and send v wei      |
 |                                             | and return the new address                                      |
-+---------------------------------------------------------------------------------------------------------------+
++---------------------------------------------+-----------------------------------------------------------------+
 | call(g:u256, a:u256, v:u256, in:u256,       | call contract at address a with input mem[in..(in+insize))      |
 | insize:u256, out:u256,                      | providing g gas and v wei and output area                       |
 | outsize:u256)                               | mem[out..(out+outsize)) returning 0 on error (eg. out of gas)   |
 | -> r:u256                                   | and 1 on success                                                |
-+---------------------------------------------------------------------------------------------------------------+
++---------------------------------------------+-----------------------------------------------------------------+
 | callcode(g:u256, a:u256, v:u256, in:u256,   | identical to ``call`` but only use the code from a              |
 | insize:u256, out:u256,                      | and stay in the context of the                                  |
 | outsize:u256) -> r:u256                     | current contract otherwise                                      |
-+---------------------------------------------------------------------------------------------------------------+
++---------------------------------------------+-----------------------------------------------------------------+
 | delegatecall(g:u256, a:u256, in:u256,       | identical to ``callcode``,                                      |
 | insize:u256, out:u256,                      | but also keep ``caller``                                        |
 | outsize:u256) -> r:u256                     | and ``callvalue``                                               |
-+---------------------------------------------------------------------------------------------------------------+
++---------------------------------------------+-----------------------------------------------------------------+
 | stop()                                      | stop execution, identical to return(0,0)                        |
-| Perhaps it would make sense retiring this as it equals to return(0,0). It can be an optimisation by the EVM   |
-| backend.                                                                                                      |
-+---------------------------------------------------------------------------------------------------------------+
+|                                             | Perhaps it would make sense retiring this as it equals to       |
```

Comment:
Where did this comment come from?

Reply:
It was there. (This is the julia part.)

Reply:
I know, but I still think this has no place in a documentation :)

Reply:
There's another PR which updates the builtins #3226, I'd just clean up these there and leave this single change as a formatting fix.